docs/en/development_handbook/quickstart.md

```diff
@@ -33,16 +64,22 @@ _NB: Take note of the email and password generated at the end of this task; you'
 
 ## Build the frontend client
 ```
-cd client && gulp compile
+gulp compile
 ```
 ## Launch the server
 ```
+cd angular
```

Comment:
Are you sure you intended to write angular? I think you want to cd to client here.

Reply:
If the user keeps the same terminal instance open they're already in client (line 54).
I think gulp changes the working directory anyway so this step may not be needed